### PR TITLE
Add testcases for ZipUtil, ZipFile, ZipArchiveOutputStream and ZipArchiveEntry

### PR DESCRIPTION
**Pull request by PeterAlfreadLee**
Add testcases for zip, including ZipUtil, ZipFile, ZipArchiveOutputStream and ZipArchiveEntry

### CODE REVIEW

File: src/test/java/org/apache/commons/compress/archivers/zip/ZipArchiveEntryTest.java [hunk 1]
```diff
@@ -22,6 +22,9 @@
 import static org.junit.Assert.*;
 
 import java.io.ByteArrayOutputStream;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
```
Comment (bodewig) [inline src/test/java/org/apache/commons/compress/archivers/zip/ZipArchiveEntryTest.java]: These are not sctually needed, are they?

File: src/test/java/org/apache/commons/compress/archivers/zip/ZipUtilTest.java [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
 import static org.junit.Assert.*;
 
+import java.io.UnsupportedEncodingException;
```
Comment (bodewig) [inline src/test/java/org/apache/commons/compress/archivers/zip/ZipUtilTest.java]: not used?

**Comment by bodewig** [comment]
Thanks @PeterAlfreadLee - I've just added a few comments about unnecessary new imports.

BTW, I think it is time you add yourself as developer to the POM. :-)

**Comment by PeterAlfreadLee** [comment]
Oops, I forgot to delete the imports when I was debugging. Sorry about that.
And I added myself as a developer in pom. :)

**Comment by bodewig** [comment]
Thank you

**Pull request merged**